py3status/modules/sysdata.py

```diff
@@ -12,7 +12,7 @@
     swap_unit: the unit of swap to use in report, case insensitive.
         ['dynamic', 'KiB', 'MiB', 'GiB'] (default 'GiB')
     temp_unit: unit used for measuring the temperature ('C', 'F' or 'K')
```

Comment:
[Satan's voice]
This kinda make more sense as `cpu_temp_unit` because it does not say which `temp` it is for... Could be `cpu`, `mem`, `ram`, or `???`. I wonder if we can obtain more with `psutil` or other tools. Although this is a bit long, it would be "correct" in the long run... We probably shouldn't change it anyway.

Alternatively, we could use `cpu_unit` to stand in line with other `*_unit`... assuming we are not going to expose CPU hertz units anytime soon... or will we? 👿 

Anyhow, I don't care. Up to you guys. Just pointing this out.